src/api/app/helpers/webui/maintenance_incident_helper.rb

```diff
@@ -0,0 +1,48 @@
+module Webui::MaintenanceIncidentHelper
+  def incident_label(incident_project, patchinfo)
+    incident_number = incident_project.name.rpartition(':').last
+    title = patchinfo.try(:[], :summary) || incident_project.title || incident_project.name
```

Comment:
[nitpick] I guess to avoid this rather strange to parse `patchinfo.try(:[], :summary)` it would be good if patchinfo was always set to at least {} in the release_targets_ng

Right now it seems to be nil, {} or a hash containing :summary depending on various states.